### ISSUE TITLE
PDFfigures 2.0 extracts tables as well as figures

### ISSUE THREAD

**Issue by jamesyiwenmcdonnell-star** [jamesyiwenmcdonnell-star/alt_text_generation#2]
PDFFIgures 2.0 is desgined to extract tables as well. This causes tables to be embedded like a figure, which breaks their tagging. Causes screen readers to be unable to properly read out the information inside the table.
